client.go

```diff
@@ -266,6 +267,7 @@ type Client struct {
 	mu         sync.RWMutex
 	url        string
 	projectID  string
+	release    string
 	authHeader string
```

Comment:
Nitpick: authHeader should abut url and projectID.

```go
url        string
projectID  string
authHeader string
release    string
```

Rationale: url, projectID, and authHeader are all set together inside `SetDSN`.